.github/doxygen.json

```diff
@@ -0,0 +1,18 @@
+{
+	"problemMatcher": [
+		{
+			"owner": "doxygen",
+			"severity": "warning",
+			"pattern": [
+				{
+					"regexp": "^(.*):(\\d+): warning: (.*)$",
+					"file": 1,
+					"line": 2,
+					"message": 3
+				}
+			]
+		}
+	]
+}
+
+
```

Comment:
Should we prefer to use spaces instead of tabs?

Reply:
Ack. Done.